Packs/Core/Integrations/CortexPlatformCore/CortexPlatformCore.py

```diff
@@ -54,6 +56,66 @@
 ASSET_COVERAGE_TABLE = "COVERAGE"
 APPSEC_RULES_TABLE = "CAS_DETECTION_RULES"
 
+ENDPOINT_TYPE = {
```

Comment:
Please put all endpoint constant in a class like done for appSecIssues and in other places.